src/components/Form/bookForm.js

```diff
@@ -26,6 +26,7 @@ function Form() {
 
   return (
     <form onSubmit={handleSubmit}>
```

Comment:
- [ ] Kindly 😊 as stated by the previous reviewer 😃 make sure to align the Form properly so it aligned with the book container, so it will be better if you stick to the original design and try to align the form with the book container to make it nice for the user like the zeplin design 
> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|   ![image](https://github.com/Booze33/book-store/assets/83360636/06ed23cd-df3c-4e80-9995-6f03ff2799a5)  |     ![image](https://github.com/Booze33/book-store/assets/83360636/a4b9d361-ea2b-4555-bf1c-86c38e3eb518)   |